### ISSUE TITLE
Wx Stations

### ISSUE THREAD

**Issue by HunterGleason** [bcgov/watershedBC#30]
Could be useful to include weather stations as points layer within and nearby the watershed, ECCC, FERN, MOTI, ASWS, others?

**Comment by bevingtona** [comment]
Good idea, also BCWS. Do you know if a single authoritative list exists? 

I also have the BCWS experimental app (https://bcgov-env.shinyapps.io/bcws_wx/) and the Northern BC research weather station app (https://bcgov-env.shinyapps.io/nbcclim/) ... would be great to amalgamate all of this into a single mega watershed app.
